tendrl/commons/utils/alert_utils.py

```diff
@@ -0,0 +1,40 @@
+import json
+import os
+
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons.utils.time_utils import now as tendrl_now
+
+
+def alert_job_status(curr_value, msg, integration_id=None, cluster_name=None):
+    alert = {}
+    alert['source'] = NS.publisher_id
+    alert['classification'] = 'cluster'
+    alert['pid'] = os.getpid()
+    alert['time_stamp'] = tendrl_now().isoformat()
+    alert['alert_type'] = 'STATUS'
+    severity = "INFO"
+    if curr_value.lower() == "failed":
+        severity = "WARNING"
+    alert['severity'] = severity
+    alert['resource'] = 'cluster'
+    alert['current_value'] = curr_value
+    alert['tags'] = dict(
+        message=msg,
+        integration_id=integration_id or
+        NS.tendrl_context.integration_id,
+        cluster_name=cluster_name or
+        NS.tendrl_context.cluster_name,
+        sds_name=NS.tendrl_context.sds_name,
+        fqdn=NS.node_context.fqdn
+    )
+    alert['node_id'] = NS.node_context.node_id
+    if not NS.node_context.node_id:
+        return
```

Comment:
@shtripat may i know why you are setting node_id for cluster related alert?

Reply:
As commented earlier by @r0h4n node_id should be made mandatory